integration/client/client_windows_test.go

```diff
@@ -57,7 +57,21 @@ func init() {
 		testImage = "mcr.microsoft.com/windows/nanoserver:2004"
 	case osversion.V20H2:
 		testImage = "mcr.microsoft.com/windows/nanoserver:20H2"
+	case 20348:
```

Comment:
@kevpar Does this seem fine for now? I didn't want to cut a new hcsshim release just to use the new osversion definitions, I wanted to wait for the tty support for host process.